code/src/tezos_baking/__init__.py

```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
```

Comment:
From the documentation:
> `__init__.py` is required to import the directory as a package, and should be empty.

so let's remove this REUSE header and instead add an entry in `.reuse/dep5`

Reply:
BTW, seems like it's not really true that `__init__.py` file should be empty, see [here](https://docs.python.org/3/tutorial/modules.html#packages).
> The __init__.py files are required to make Python treat directories containing the file as packages. This prevents directories with a common name, such as string, from unintentionally hiding valid modules that occur later on the module search path. In the simplest case, __init__.py can just be an empty file, but it can also execute initialization code for the package or set the __all__ variable, described later.

So example, it's idiomatic way to set package's version in code, see [here](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/).

Reply:
Well, sure, `__init__.py` can be non-empty, but it should be if we follow the documentation (especially for the simplest cases, like our own).

> it's idiomatic way to set package's version in code, see here.

I wouldn't call that the idiomatic way, at least not anymore, given that:
1. it's the 6th option in that list
2. it's the only one warning that it will fail to work in some cases 

Given that we are currently matching version number with the package, I think we can get the version from the same `meta.json` that we use everywhere else (which is kinda what option 4 is BTW).
However TBH I don't see any need to do this right now, as we don't need that version number in the wizard's code at all and this info is just metadata at this point.

So, IMO this change should be reverted and once again `__init__.py` left empty.

P.s. even if we did need to set a version there, it shouldn't be `v17.1` as that's the version of Octez (that we don't need to match, as it's unrelated to this) and not the package version (that's `17.1-1` and we also don't need to match that).